developer-docs-site/docs/guides/sponsored-transactions.md

```diff
@@ -0,0 +1,70 @@
+# Sponsored Transactions
+
+As outlined in [AIP-39](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-39.md), a sponsored transactions allow a blockchain account to pay the fees associated with executing a transaction for another account. Sponsored transactions simplify the process for onboarding users into applications by allowing the application to cover all associated fees for interacting with the Aptos blockchain. Here are two examples:
+* [MerkleTrade](https://merkle.trade/) offers low cost trading to those with Ethereum wallets by creating an Aptos wallet for users and covering all transaction fees so that the user does not need to acquire utility tokens for Aptos.
+* Community engagement applications like [Graffio](https://medium.com/aptoslabs/graffio-web3s-overnight-sensation-81a6cf18b626) offered to cover transaction fees for custodial accounts to support the collaborative drawing application for those without wallets.
+
+## Process Overview
+
+The process for sending a sponsored transaction follows:
+* The sender(s) of the transaction agree upon an operation, as defined by a `RawTransaction`.
+* The sender(s) generate a `RawTransactionWithData::MultiAgentWithFeePayer` structure
+  * As noted below, prior to the framework 1.8 release, this must contain the fee payer's address.
+  * After framework release 1.8, this can optionally be set to `0x0`.
+* The sender(s) can forward the signed transaction to the fee payer who signs with their signature and forwards it to the blockchain.
+* Upon execution of the transaction, the sequence number of the primary sending account is incremented and all gas fees are deducted and all refunds are sent to the gas fee payer.
+
+Alternatively, if the fee payer knows the operation and all parties involved, they could sign a transaction and forward it back to the parties to sign.
```

Comment:
Parties is fine here, though I think we should probably stick with common terms, such as signers, rather than move between different terms.

```suggestion
Alternatively, if the fee payer knows the operation and all signers involved, the fee payer could generate and sign the transaction and send it to the other signers to sign.
```